src/main/java/uk/gov/pay/adminusers/validations/RequestValidations.java

```diff
@@ -58,6 +64,14 @@
         }
         return errors.isEmpty() ? Optional.empty() : Optional.of(errors);
     }
+    
+    public Optional<List<String>> isValidEnumValue(JsonNode payload, EnumSet enumSet, String field) {
+        String value = payload.get(field).asText();
+        if (enumSet.stream().noneMatch(constant -> constant.toString().equals(value))) {
```

Comment:
It would be safer to use `.name()` here because it will always give the exact name, whereas `toString()` can be overridden (you’ll have to add the generic type parameter detailed in my other comment for this to work)